### ISSUE TITLE
Feature Request - Facedetailer output node / flow

### ISSUE THREAD

**Issue by lanthos** [SeargeDP/SeargeSDXL#72]
I love how nice and clean everything is in the workflows but, especially now with version 4, it can be really hard to add in other flows.  Since you have everything going through your custom data flow there's no way to pull out all of the things one would need to use FaceDetailer from the ComfyUI-Impact-Pack.  How hard would it be to get a node that would take the data stream and output: model, clip, vae, positive prompt, and negative prompt? 

Or have you considered adding something like FaceDetailer/Adetailer into your flows?

Thank you for the consideration!  And for the work you've put into this already, it's really nice and helps the community a ton.

**Comment by SeargeDP** [comment]
It's planned for a future update to make it easier to hook into the flow of the nodes and add custom things.

**Comment by TheForgotten69** [comment]
That would be so nice. This pack is simply the best I ever used for Comfy

**Comment by Arkhaner1511** [comment]
Yeah a brakeout node would be perfect. Ideally, all parameters of data should be optionally selectable.

**Comment by Maxxxel** [comment]
yes please

**Comment by timoshishi** [comment]
Face Detailer/Adetailer are definitely needed!

**Comment by rafstahelin** [comment]
second
